src/finch/tensor.py

```diff
@@ -809,6 +883,30 @@ def atan2(x: Tensor, other: Tensor, /) -> Tensor:
     return x._elemwise_op("atand", other)
 
 
+def bitwise_and(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1 & x2
+
+
+def bitwise_or(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1 | x2
+
+
+def bitwise_left_shift(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1 << x2
+
+
+def bitwise_right_shift(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1 >> x2
+
+
+def bitwise_xor(x1: Tensor, x2: Tensor, /) -> Tensor:
+    return x1 ^ x2
+
+
+def bitwise_invert(x: Tensor, /) -> Tensor:
+    return ~x
```

Comment:
Can we replace these with `from operator import ... [as ...]`?

Reply:
While we're at it, let's do it for all functions where this may apply.

Reply:
Done! (it reduced a bit of boilerplate code)